src/main/java/com/gigacal/service/impl/CalendarServiceImpl.java

```diff
@@ -25,4 +31,14 @@ public void createCalendar(final CalendarEntity calendarEntity) {
         calendarEntity.setCreateDate(LocalDateTime.now());
         calendarRepository.save(calendarEntity);
     }
+
+    @Override
+    public void validateThatCalendarBelongsToUser(final Long calendarId, final Long userId) {
+        final Optional<CalendarEntity> optionalCalendarEntity = this.calendarRepository.findByIdAndUserId(calendarId, userId);
+
+        if (optionalCalendarEntity.isEmpty()) {
+            LOGGER.warn("Calendar with id={} doesn't belong to user with id={}", calendarId, userId);
+            throw new ForbiddenActionException("Calendar with id=" + calendarId + " doesn't belong to user with id=" + userId);
```

Comment:
Nope - I'm highly conviced that we should log any attempts that some user tries to do something forbidden in our system. Therefore, using an orElseThrow would require me to do lambda expression inside of it which looks rather weird to me + it violates this rule - https://rules.sonarsource.com/java/RSPEC-2201

I've changed it so that there's no Optional value saved as a variable